anchore_engine/services/catalog/__init__.py

```diff
@@ -414,10 +411,8 @@ def handle_service_watcher(*args, **kwargs):
         logger.debug("FIRING: service watcher")
 
         localconfig = anchore_engine.configuration.localconfig.get_config()
-        verify = localconfig["internal_ssl_verify"]
 
         with db.session_scope() as dbsession:
-            mgr = manager_factory.for_session(dbsession)
```

Comment:
mgr wasn't being used. doesn't look like the identity manager does anything on instantiation